rust/processor/migrations/2023-11-09-234725_delegator_balances_2/down.sql

```diff
@@ -0,0 +1,6 @@
+-- This file should undo anything in `up.sql`
```

Comment:
Not dropping `delegator_balances` table here?

Reply:
yeah nah, no worries; we wont